src/main/java/application/rest/v1/actions/AppPodlistFunction.java

```diff
@@ -68,7 +69,7 @@ public String invoke(ResolutionContext context, List<String> parameters) {
         else {
             // parameters.get(0) :: appNamespace
             // parameters.get(1) :: appName
-            final String appNamespace = parameters.get(0);
+            appNamespace = parameters.get(0);
```

Comment:
You're only retrieving the namespace in the else branch. It's null on the if branch.